Monika After Story/game/updates.rpy

```diff
@@ -377,7 +377,20 @@ label v0_3_1(version=version): # 0.3.1
 # 0.12.5
 label v0_12_5(version="v0_12_5"):
     python hide:
-        pass
+        local_now = mas_utils.utc_to_local(datetime.datetime.utcnow())
```

Comment:
```suggestion
        local_now = datetime.datetime.now()
```

utc APIs are for converting existing times, can just use regular now for local now